src/pages/Admin/Admin_v2.tsx

```diff
@@ -140,15 +139,13 @@ const Admin_v2: React.FC = () => {
   };
 
   return (
-    <>
-      <S.Wrapper hasSelectedVenue={!!selectedVenue}>
-        {selectedVenue && (
-          <AdminSidebar
-            sidebarOptions={sidebarOptions}
-            selected={selectedOption}
-            onClick={setSelectedOption}
-          />
-        )}
```

Comment:
Looks like the new code doesn't have this `selectedVenue` check/styles anymore? Whys that?

Reply:
Added back.